debian/base/include/etc/confluent/docker/mesos-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+if [ -z $SKIP_MESOS_AUTO_SETUP ]; then
```

Comment:
How come u need $SKIP.. even though you are already checking MESOS_SANDBOX? As long as the image works when MESOS_SANDBOX is not set, then that seems safe.

Reply:
yes it's safe for all non-users of mesos. But the idea here is that a small minority of mesos users may still prefer a manual configuration than an out of the box experience. I'm accommodating for that case.